PIL/TiffTags.py

```diff
@@ -130,16 +130,17 @@ def lookup(tag):
     324: ("TileOffsets", LONG, 0),
     325: ("TileByteCounts", LONG, 0),
 
+    330: ("SubIFD", SHORT, 1),
     332: ("InkSet", SHORT, 1),
     333: ("InkNames", ASCII, 1),
     334: ("NumberOfInks", SHORT, 1),
     336: ("DotRange", SHORT, 0),
     337: ("TargetPrinter", ASCII, 1),
     338: ("ExtraSamples", SHORT, 0),
-    339: ("SampleFormat", SHORT, 0),
+    339: ("SampleFormat", SHORT, 1),
 
-    340: ("SMinSampleValue", DOUBLE, 0),
-    341: ("SMaxSampleValue", DOUBLE, 0),
+    340: ("SMinSampleValue", DOUBLE, 1),
+    341: ("SMaxSampleValue", DOUBLE, 1),
```

Comment:
I don't see why these changes that were made -
https://web.archive.org/web/20240329145340/https://www.awaresystems.be/imaging/tiff/tifftags/sampleformat.html
https://web.archive.org/web/20240329145243/https://www.awaresystems.be/imaging/tiff/tifftags/sminsamplevalue.html
https://web.archive.org/web/20240329145243/https://www.awaresystems.be/imaging/tiff/tifftags/smaxsamplevalue.html